modules/homepage/src/main/frontend/src/themePage/Sidebar/sidebarStyle.jsx

```diff
@@ -169,6 +169,10 @@ import {
     whiteFont: {
       color: whiteColor
     },
+    versionText: {
+      textAlign: "center",
+      color: whiteColor
```

Comment:
The text needs to breathe a little. With long version numbers and/or long app names it gets too close to the edges, and it's also eclipsed by the "Administration" entry above it.
```suggestion
      color: whiteColor,
      margin: theme.spacing(4, 2, 0),
```

Reply:
Resolved in `be76129` :heavy_check_mark: